tests/unit/test_adapter.py

```diff
@@ -265,7 +265,9 @@ def test_certificate_authentication(self):
             }
         )
         credentials = connection.credentials
+        trino_auth = credentials.trino_auth()
         self.assertIsInstance(credentials, TrinoCertificateCredentials)
+        self.assertIsInstance(trino_auth, trino.auth.CertificateAuthentication)
```

Comment:
We can as well assert equality.

`self.assertEqual(credentials.trino_auth(), ....)`

Reply:
added 👍